software/controller/lib/debug/debug_types.h

```diff
@@ -37,17 +37,20 @@ enum class ErrorCode : uint8_t {
   kInternalError = 0x05,   // Some type of internal error (aka bug)
   kUnknownVariable = 0x06, // The requested variable ID is invalid
   kInvalidData = 0x07,     // data is out of range
+  kWait = 0x08,            // awaiting peripheral response
```

Comment:
Is waiting a really an error? It seems to be a non-error state, which is also defined above as a state?